test/base/test_collector.py

```diff
@@ -200,7 +200,6 @@ def test_collector_with_dict_state():
     assert not np.isclose(obs[0]['rand'], obs[1]['rand'])
     c1 = Collector(policy, envs, ReplayBuffer(size=100),
                    Logger.single_preprocess_fn)
-    c1.seed(0)
```

Comment:
Why to remove this ?

Reply:
Because seed should be done in env instead of collector. Same as render.